### PR TITLE
Added Create Pull request section as per the requirement

### LINKED ISSUE #3165: [PR] Add a section to file on how to open new PR for newcomers
**Current Behavior**

As dicussed in the previous meet, many newcomers face the issue of getting started . So this will help them in getting started.
**Desired Situation**

1> A small section needs to be added on how to open your first PR. 

---
[Optional] **Alternatives**

[Optional] **Additional context**

file -> https://layer5.io/community/handbook/contribution
---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by sam1code**
Signed-off-by: Soumyakanta Pattanaik <samcse95@gmail.com>

**Description**
Screenshot from 2022-10-22 03-30-45

This PR fixes #3165

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1ec8097da8cfe6f2abca96b6379a697d261b5fe0 at: https://635319e7c6297100b83d81b2--layer5.netlify.app

**Review by iambami — APPROVED** [review]
Looks good. Thanks @sam1code

File: src/sections/Community/Handbook/contributing.js [hunk 1]
```diff
@@ -146,6 +150,30 @@ const contributingGuide = () => {
...
+                  <StaticImage src={ compareAndPR }  alt="Compare and pull request" />
+                </div>
+              </li>
+            </ul>
+            <ul>
+              <li>
+                <span>
+                  Once you click on the button, you'll be taken to the Pull Request page. Here, you can add a title and description to your pull request that explains your contribution. Once you're done, click on the Create pull request button.
+                </span>
+                <div>
```
Comment (legendarykamal) [inline src/sections/Community/Handbook/contributing.js:170]: A note should also be added in here about check if the right branches are being compared for which the PR is to be made. 

PS:- As in some cases it doesn't , and the PR results with the users forked repo master branch itself.

**Review by legendarykamal — CHANGES_REQUESTED** [review]
@sam1code looks good but ideally this section should come after the `sign-off` section

**Comment by sam1code** [comment]
Ok @legendarykamal I'm changing in some time thankyou

File: src/sections/Community/Handbook/contributing.js (hunk 1 above)
Comment (sam1code) [reply src/sections/Community/Handbook/contributing.js:170]: Thankyou I'll add this as well

**Comment by deepeshaburse** [comment]
LGTM, thanks @sam1code!

**Review by deepeshaburse — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit e322efcfc404ed8bdcfb4e823a098ef3851884fe at: https://6357c063b2fdb84611a36e35--layer5.netlify.app

**Comment by sam1code** [comment]
I hope this met it's requirement!

**Comment by sam1code** [comment]
I hope all the requirements satisfied !

**Comment by l5io** [comment]
🚀 Preview for commit e6eefbbb87a35c1b91892e7f0d0c313cc2801811 at: https://6357c863d1c48f471f05a15a--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 28c6d3aea42c33ed73b037ec81823f2f727ba7c2 at: https://63590e70de26390fc767b417--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**